source/src/in/joind/AddComment.java

```diff
@@ -38,7 +38,8 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         // Allow ActionBar 'up' navigation
-        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        ActionBar actionBar = getSupportActionBar();
+        if (actionBar != null) actionBar.setDisplayHomeAsUpEnabled(true);
```

Comment:
It can happen when the Theme changes, for example. But the reason why I made the change was because I was cleaning the code with the help of Lint, and it was throwing a warning there.